<!DOCTYPE HTML>
<!--
	Prologue by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Dr Binghao Chai</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		
		<link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
		<link rel="shortcut icon" href="/favicon.ico" />
		<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
		<meta name="apple-mobile-web-app-title" content="Dr Binghao Chai" />
		<link rel="manifest" href="/site.webmanifest" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<div id="header">

				<div class="top">

					<!-- Logo -->
						<div id="logo">
							<!-- <span class="image avatar48"><img src="images/avatar.jpg" alt="" /></span> -->
							<h1 id="title">Dr Binghao Chai</h1>
							<p>Computer Vision Scientist</p>
						</div>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="#top" id="top-link"><span class="icon solid fa-home">Home</span></a></li>
								<li><a href="#about" id="about-link"><span class="icon solid fa-user">About Me</span></a></li>
								<li><a href="#research" id="research-link"><span class="icon solid fa-th-list">Research</span></a></li>
								<li><a href="#teaching" id="teaching-link"><span class="icon solid fa-book">Teaching and Supervision</span></a></li>
								<li><a href="#miscellaneous" id="miscellaneous-link"><span class="icon solid fa-archive">Miscellaneous</span></a></li>
								<!-- <li><a href="#contact" id="contact-link"><span class="icon solid fa-envelope">Contact</span></a></li> -->
							</ul>
						</nav>

				</div>

				<div class="bottom">

					<!-- Social Icons -->
						<ul class="icons">
							<!-- <li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li> -->
							<!-- <li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li> -->
							<li><a target="_blank" href="https://github.com/cbhindex" class="icon brands fa-github"><span class="label">Github</span></a></li>
							<!-- <li><a href="#" class="icon brands fa-dribbble"><span class="label">Dribbble</span></a></li> -->
							<li><a target="_blank" href="https://www.linkedin.com/in/cbhindex/" class="icon brands fa-linkedin"><span class="label">LinkedIn</span></a></li>
							<li><a href="mailto:b.chai@ucl.ac.uk" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
						</ul>

				</div>

			</div>

		<!-- Main -->
			<div id="main">

				<!-- Intro -->
					<section id="top" class="one dark cover">
						<div class="container">

							<header>
								<h2 class="alt" style="background-color: rgba(169, 169, 169, 0.4)">Hi! I'm <strong>Binghao</strong>, a research fellow in Artificial Intelligence (AI) for healthcare at <a target="_blank" href="https://www.ucl.ac.uk/cancer/welcome-ucl-cancer-institute">UCL Cancer Institute</a>.</h2>
							</header>

							<footer>
								<a href="#about" class="button scrolly">See more</a>
							</footer>

						</div>
					</section>

				<!-- About Me -->
					<section id="about" class="three">
						<div class="container">

							<header>
								<h2>About Me</h2>
							</header>
							
							<div class="row aln-center aln-middle">
								<div class="col-5 col-9-mobile">
									<article class="item">
										<a class="image fit"><img src="images/binghao_chai.jpg" alt="Dr Binghao Chai at his PhD graduation ceremony." /></a>
										<header>
											<h6>Dr Binghao Chai at his PhD graduation ceremony.</h6>
										</header>
									</article>
								</div>
							</div>
							
							<br />

							<p align="left"> My name is Binghao Chai (柴Ch&#225i, 秉浩B&#464ng H&#224o), I am a research fellow in Artificial Intelligence (AI) for healthcare at UCL Cancer Institute (<a target="_blank" href="https://www.ucl.ac.uk/cancer/research/department-pathology/genetics-and-cell-biology-sarcoma">Flanagan Lab</a>). Prior to this, I worked at Queen Mary University of London (QMUL) as a postdoctoral computer vision scientist at <a target="_blank" href="http://www.draviamlab.uk/">Draviam Lab</a> under the Innovation UK Knowledge Transfer Partnership (KTP) scheme with ZEISS. I was awarded my PhD degree in Computer Science and Digital Pathology from University College London (UCL) under the supervision of <a href="https://www.gla.ac.uk/schools/computing/staff/kevinbryson/" target="_blank">Dr Kevin Bryson</a> and <a href="https://www.ucl.ac.uk/cancer/research/department-pathology/sarcoma-biology-and-genomics" target="_blank">Dr Nischalan Pillay</a>.</p>
							
							<br />
								
							<p align="left"> My research interests are applied-AI/ML, biomedical imaging, computational pathology and cell biology imaging analysis. I take an entrepreneurial approach to my career and all single work, and I have driven change and innovation in every role by focusing on key goals and applying a growth mindset.</p>

						</div>
					</section>

				<!-- research -->
					<section id="research" class="two">
						<div class="container">

							<header>
								<h2>Research</h2>
							</header>

							<h3 align="left"><strong>Exploring the impact of Haematoxylin and Eosin staining variability on sarcoma subtypes across multiple laboratories using Artificial Intelligence (AI)-based methods</strong></h3>
							
							<p align="left">We have an exciting new collaboration with Google Health to explore sarcoma subtyping and the impact of colour variability across multiple laboratories in sarcoma pathology. This is accomplished by applying a combination of the Google Path Foundation model and other machine learning techniques on digitised histological slides to unravel the unique architectural and cytological nuances of these enigmatic tumours for the benefit of diagnosis. Our first pilot project is currently being undertaken by Dr Binghao Chai in collaboration with Prof. Adrienne Flanagan (UCL), Dr Tapabrata Chakraborty (The Alan Turing Institute) and Dr David Steiner (Google Health). Read more about the Path Foundation model <a href="https://research.google/blog/health-specific-embedding-tools-for-dermatology-and-pathology/" target="_blank" style="color: blue">here</a>, and their preprint <a href="https://arxiv.org/abs/2310.13259" target="_blank" style="color: blue">here</a>.</p>
							
							<br />
							
							<h3 align="left"><strong>Multi-SpinX: an advanced framework for automated tracking of mitotic spindles and kinetochores in multicellular environments</strong></h3>
							
							<p align="left">With the advent of advanced imaging technologies, research groups routinely generate terabytes of live-cell movies every year. Analysing these movies manually is not only tedious, it is also a waste of experienced researchers. Recognising this problem, several automated image analysis software tools are being developed but no full proof solution exists for high-resolution live-movies of dynamic subcellular structures that move in 3D. This is a particularly unique problem because of the temporal and spatial challenges in imaging along with phototoxicity which reduce the amount of information available for reliable automated analysis. Draviam Lab has been able to overcome this using a combination of guided modelling and deep learning methods developed to track mitotic spindles in 3-Dimensions (3D) through time (SpinX). </p>
							
							<br />
							
							<div class="row aln-center aln-middle">
								<div class="col-10 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/spinx_demo.gif" alt="SpinX demo." /></a>
										<header>
											<h6>SpinX demo.</h6>
										</header>
									</article>
								</div>
							</div>
							
							<br />
							
							<p align="left">During mitosis, the microtubules of the mitotic spindle are tethered to chromosomes by a specialised macromolecular structure, the kinetochore that assembles on the centromeric region of chromosomes. The mitotic spindle is constantly changing shape and it undergoes movements independent from the kinetochores that are also moving within the spindle. The tracking challenge becomes computationally intensive due to either the crowding of objects within cells or the high-throughput nature of the study presenting many dividing cells next to each other. SpinX was limited to analysing single cells and spindles in metaphase. In response to the growing demand for a more adaptable tool that is suitable for high-throughput studies, we have now developed a framework, termed Multi-SpinX, to enhance the capabilities of SpinX. Multi-SpinX represents a significant expansion of our original framework, and is able to track spindles and kinetochores in a multiple-cell environment. This work was led by Dr Binghao Chai and was conducted in collaboration with Prof. Viji Draviam (Queen Mary University of London, QMUL), Prof Kozo Tanaka (Tohoku University), Christoforos Efstathiou (QMUL), Muntaqa Choudhury (QMUL), Kinue Kuniyasu (Tohoku University) and ZEISS. This project was funded by the Innovation UK KTP scheme and ZEISS. This research is available <a href="https://www.biorxiv.org/content/10.1101/2024.04.03.587736v3.abstract" target="_blank" style="color: blue">here</a>. Other related publications are available <a href="https://doi.org/10.1016/j.tcb.2023.10.010" target="_blank" style="color: blue">here</a> (review paper, Trends in Cell Biology) and <a href="https://www.rms.org.uk/resource/ai-for-advanced-image-analysis.html" target="_blank" style="color: blue">here</a> (interview article, infocus Magazine from Royal Microscopical Society).</p>
							
							<br />
							
							<div class="row aln-center aln-middle">
								<div class="col-10 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/multi_spinx.jpg" alt="Multi-SpinX." /></a>
										<header>
											<h6>Time-lapse images of consecutive frames illustrate the complex process of kinetochore tracking overlaid with spindle tracking. The frames are organised as follows: the original movie as a merge of the spindle (red) and kinetochore marker (centromere marker for chromosome 16, in green) displays the representative consecutive frames (first row), the tracked spindle with bounding boxes illustrates the spindle tracking outcome (second row), the tracked GFP with bounding boxes (red, marked by yellow arrow) depicts the kinetochores being tracked, with bounding boxes delineating the track location (third row), and the magnified consecutive frames highlighted with yellow arrows in the third row showcase sister kinetochores with two particle IDs (1 and 2) segregated into four kinetochores (1,2,3 and 4). Scale bar: 10 μm.</h6>
										</header>
									</article>
								</div>
							</div>
							
							<br />
							
							<h3 align="left"><strong>Using machine learning techniques on digitised histopathological images to distinguish lipoma (LP) from atypical lipomatous tumours (ALT)</strong></h3>
							
							<p align="left">Lipomas are benign neoplasms of fat and are amongst the most common tumours with an estimated incidence rate of 1 per 1000 people per year (Rydholm &amp Berg 1983, Weiss et al. 2007). They need to be distinguished from atypical lipomatous tumours (ALT) which are malignant and rare and present both a clinical and histological challenge. The distinction is made by assessing nuclear and subtle architectural features which require reviewing multiple sections and the use of ancillary genetic testing. Benign fatty tumours, therefore, present a considerable workload in the general pathology setting and the distinction from malignancy often require specialist review. This problem can be addressed by automated whole slide image (WSI) analysis. However, the paucicellular nature of the fatty tissues also presents a computational challenge.</p>
							
							<br />
							
							<div class="row aln-center aln-middle">
								<div class="col-10 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/lp_alt_1.jpg" alt="Septa and the nuclei along with septa." /></a>
										<header>
											<h6>Septa and the nuclei along with septa in an atypical lipomatous tumour sample.</h6>
										</header>
									</article>
								</div>
							</div>
							
							<br />
							
							<p align="left">In order to explore the utility of machine learning and deep learning techniques on fatty tumour digital pathology for the diagnosis benefits, we developed and validated a computational pipeline for distinguishing these two types of fatty tumours at the slide-level. An ideal automatic pipeline should be able to predict the conventional lipomas with 100% accuracy with all other cases going to be manually reviewed including those unusual lipomas and the atypical lipomatous tumours. Although our research findings are not the case, they indicate that computational approaches have some ability to distinguish malignant atypical lipomatous tumours from benign lipomas just based on their histopathological images. In terms of the generalisability, the performance of this pipeline decreases when applied to the slides from different external labs. This work was led by Dr Binghao Chai and was conducted in collaboration with Dr Kevin Bryson and Dr Nischalan Pillay. This research is available <a href="https://discovery.ucl.ac.uk/id/eprint/10170583/" target="_blank" style="color: blue">here</a>.</p>
							
							<br />
							
							<div class="row aln-center aln-middle">
								<div class="col-10 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/lp_alt_2.jpg" alt="Atypical lipomatous tumour and its deep learning inference." /></a>
										<header>
											<h6>An atypical lipomatous tumour sample and its inference result in heat map. In the heat map, the malignant scores are descendingly sort in colours of red, orange, green and blue.</h6>
										</header>
									</article>
								</div>
							</div>
							
							<br />

						</div>
					</section>
				
				<!-- teaching -->
					<section id="teaching" class="two">
						<div class="container">

							<header>
								<h2>Teaching and Supervision</h2>
							</header>
							
							<h3 align="left"><strong>Research Project Supervision</strong></h3>
							
							<p align="left">Research project supervisor to <u>Muntaqa Shahana Choudhury</u> (PhD candidate at QMUL), <u>Saakshi Sanjay Jain</u> and <u>Sana Zakir Piracha</u> (both undergrad students at QMUL) for the Multi-SpinX project (2024).</p>
							
							<br />
							
							<h3 align="left"><strong>Master's and Bachelor's Thesis Supervision</strong></h3>
							
							<p align="left"><u>Alexia-Cristina Maharea</u>, bachelor's thesis at QMUL (2024). Thesis: Assessing multi-spindle tracker to improve and support development.</p>
							<p align="left"><u>Alan Jeyaram Sounthararajah</u>, bachelor's thesis at QMUL (2024). Thesis: Improving SpinX: the multispindle tracker.</p>
							<p align="left"><u>Jeel Maheshkumar Prajapati</u>, master's thesis at QMUL (2023). Thesis: Artificial Intelligence tools to advance drug discovery screens: developing a nuclear atypia tracker using a deep learning framework at Zeiss arivis Cloud.</p>
							
							<br />
							
							<h3 align="left"><strong>Teaching Assistant</strong></h3>
							
							<p align="left"><u>Requirement Engineering and Software Architecture (COMP0101)</u> at UCL (2021/22, 2020/21, 2019/20, 2018/19).</p>
							<p align="left"><u>Software Abstraction and Systems Integration (COMP0102)</u> at UCL (2021/22, 2020/21, 2019/20, 2018/19).</p>
							<p align="left"><u>Validation and Verification (COMP0103)</u> at UCL (2020/21).</p>
							<p align="left"><u>Systems Engineering (COMP0016)</u> at UCL (2020/21).</p>
							<p align="left"><u>Machine Learning for Domain Specialists (COMP0142)</u> at UCL (2019/20).</p>
							<p align="left"><u>Software Enginnering (COMP0071)</u> at UCL (2018/19).</p>							
							
				<!--		<div class="row">
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a href="#" class="image fit"><img src="images/pic02.jpg" alt="" /></a>
										<header>
											<h3>Ipsum Feugiat</h3>
										</header>
									</article>
									<article class="item">
										<a href="#" class="image fit"><img src="images/pic03.jpg" alt="" /></a>
										<header>
											<h3>Rhoncus Semper</h3>
										</header>
									</article>
								</div>
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a href="#" class="image fit"><img src="images/pic04.jpg" alt="" /></a>
										<header>
											<h3>Magna Nullam</h3>
										</header>
									</article>
									<article class="item">
										<a href="#" class="image fit"><img src="images/pic05.jpg" alt="" /></a>
										<header>
											<h3>Natoque Vitae</h3>
										</header>
									</article>
								</div>
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a href="#" class="image fit"><img src="images/pic06.jpg" alt="" /></a>
										<header>
											<h3>Dolor Penatibus</h3>
										</header>
									</article>
									<article class="item">
										<a href="#" class="image fit"><img src="images/pic07.jpg" alt="" /></a>
										<header>
											<h3>Orci Convallis</h3>
										</header>
									</article>
								</div>
							</div> -->

						</div>
					</section>

				<!-- miscellaneous -->
					<section id="miscellaneous" class="two">
						<div class="container">

							<header>
								<h2>Miscellaneous</h2>
							</header>
							
							<h3 align="left"><strong>Upcoming news</strong></h3>
							
							<ul>
							
								<li><p align="left">I will present <i>Exploring the impact of Haematoxylin and Eosin staining variability on sarcoma subtypes across multiple laboratories using Artificial Intelligence (AI)-based methods</i> at <a href="https://www.pathsoc.org/events/84/2025_winter_meeting/" target="_blank">Pathological Society 2025 Joint Winter Meeting</a> with The Royal Society of Medicine (Jan 21<sup>st</sup>, 2025).</p></li>
							
							</ul>
							
							<br/>
							
							<h3 align="left"><strong>Binghao and his classic music</strong></h3>
							
							<div class="row">
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a href="https://www.youtube.com/watch?v=a3YM7eXCzMU" target="_blank" class="image fit"><img src="images/beethoven_pathetique.jpg" alt="Dr Binghao Chai Plays Beethoven, Adagio Cantabile (from Sonata Pathetique No.8 Op.13)" /></a>
										<header>
											<h3>Dr Binghao Chai Plays Beethoven, Adagio Cantabile (from Sonata Pathetique No.8 Op.13)</h3>
										</header>
									</article>
								</div>
							</div> 
							
							<br/>
							
							<h3 align="left"><strong>Gallery</strong></h3>
							
							<div class="row">
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/Binghao_draviam_lab.JPG" alt="Binghao and Draviam Lab" /></a>
										<header>
											<h3>Binghao and Draviam Lab (2024)</h3>
										</header>
									</article>
									<article class="item">
										<a class="image fit"><img src="images/Binghao_draviam_lab_2.JPG" alt="Binghao and Draviam Lab 2" /></a>
										<header>
											<h3>Binghao and Draviam Lab (2024)</h3>
										</header>
									</article>
								</div>
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/binghao_viva.jpg" alt="Binghao on his PhD Viva" /></a>
										<header>
											<h3>Binghao on his PhD viva (examinars: Prof. Mohammad Ilyas and Dr. Charles-Antoine Collins Fekete, 2023)</h3>
										</header>
									</article>
								</div>
								<div class="col-4 col-12-mobile">
									<article class="item">
										<a class="image fit"><img src="images/binghao_ecdp_2019.jpg" alt="Binghao on ECDP 2019" /></a>
										<header>
											<h3>Binghao and his PhD supervisors (Dr Kevin Bryson and Dr Nischalan Pillay) on the 15th European Congress on Digital Pathology (2019)</h3>
										</header>
									</article>
								</div>
							</div>

							<br/>
							
							<h3 align="left"><strong>Old News</strong></h3>
							
							<ul>
							
								<li><p align="left">Jul. 2024, I co-hosted "<a href="http://www.draviamlab.uk/london_workshop_2024_July.html" target="_blank">International Symposium on Genomic Instability and Impact of Genetic Variants</a>" at Queen Mary University of London (London, UK) and presented in a poster for "Multi-SpinX: An Advanced Framework for Automated Tracking of Mitotic Spindles and Kinetochores in Multicellular Environments".</p></li>
								<li><p align="left">Apr. 2024, I joined <a href="https://www.ucl.ac.uk/cancer/research/department-pathology/genetics-and-cell-biology-sarcoma" target="_blank">Flanagan Lab</a> at UCL Cancer Institute as a research fellow in AI for healthcare.</p></li>
								<li><p align="left">Mar. 2024, I co-hosted "<a href="http://www.draviamlab.uk/london_workshop_2024_March.html" target="_blank">Cell Dynamics and Chromosomal Stability Workshop</a>" at Queen Mary University of London (London, UK) and presented "Multi-SpinX: An Advanced Framework for Automated Tracking of Mitotic Spindles and Kinetochores in Multicellular Environments" in the meeting.</p></li>
								<li><p align="left">Dec. 2023, I co-hosted "<a href="http://www.draviamlab.uk/london_workshop_2023.html" target="_blank">Cell Dynamics and Chromosomal Stability Workshop</a>" at Queen Mary University of London (London, UK) and presented "AI in biomedical image and movie analysis" in the meeting.</p></li>
								<li><p align="left">Sep. 2023, I co-hosted "<a href="http://www.draviamlab.uk/beijing_workshop.html" target="_blank">Cell Dynamics and Chromosomal Stability Workshop</a>" at China Agricultural University (Beijing, China) and presented "AI in biomedical image and movie analysis" in the meeting.</p></li>
								<li><p align="left">Aug. 2023, I co-hosted "<a href="https://www.he.kanagawa-it.ac.jp/~hiroi/celldyna.html" target="_blank">Cell Dynamics and Chromosomal Stability Workshop</a>" at Kanagawa Institute of Technology (Tokyo, Japan) and presented "AI in biomedical image and movie analysis" in the meeting.</p></li>
								<li><p align="left">Mar. 2023, I passed my final viva and was awarded a PhD degree.</p></li>
								<li><p align="left">Jan. 2023, I joined <a href="http://www.draviamlab.uk/" target="_blank">Draviam Lab</a> at Queen Mary University of London as a postdoctoral computer vision scientist.</p></li>
								<li><p align="left">Jul. 2019, I passed my upgrade viva and became a PhD candidate.</p></li>
								<li><p align="left">Apr. 2019, I presented "An emerging role for machine learning in the classification of tumours of fat: distinguishing benign from malignant" at the 15th European Congress on Digital Pathology (ECDP 2019).</p></li>
								<li><p align="left">Jan. 2018, I joined <a href="https://www.gla.ac.uk/schools/computing/staff/kevinbryson/" target="_blank">Bryson Lab</a> in collobration with <a href="https://www.ucl.ac.uk/cancer/research/department-pathology/sarcoma-biology-and-genomics" target="_blank">Pillay Lab</a> at UCL as a PhD student.</p></li>
							
							</ul>

						</div>
					</section>
				
<!--
				<!-- Contact ->
					<section id="contact" class="four">
						<div class="container">

							<header>
								<h2>Contact</h2>
							</header>

							<p>Elementum sem parturient nulla quam placerat viverra
							mauris non cum elit tempus ullamcorper dolor. Libero rutrum ut lacinia
							donec curae mus. Eleifend id porttitor ac ultricies lobortis sem nunc
							orci ridiculus faucibus a consectetur. Porttitor curae mauris urna mi dolor.</p>

							<form method="post" action="#">
								<div class="row">
									<div class="col-6 col-12-mobile"><input type="text" name="name" placeholder="Name" /></div>
									<div class="col-6 col-12-mobile"><input type="text" name="email" placeholder="Email" /></div>
									<div class="col-12">
										<textarea name="message" placeholder="Message"></textarea>
									</div>
									<div class="col-12">
										<input type="submit" value="Send Message" />
									</div>
								</div>
							</form>

						</div>
					</section> 
-->
				
			</div>

		<!-- Footer -->
			<div id="footer">

				<!-- Copyright -->
					<ul class="copyright">
						<li>&copy; 2024 Dr Binghao Chai. All rights reserved.</li><!-- <li>Design: <a href="http://html5up.net">HTML5 UP</a></li> -->
					</ul>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>